burn-no-std-tests/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+authors = [
+  "nathanielsimard <nathaniel.simard.42@gmail.com>",
+  "Dilshod Tadjibaev (@antimora)",
+]
+edition = "2021"
+license = "MIT/Apache-2.0"
+name = "burn-no-std-tests"
+readme = "README.md"
+repository = "https://github.com/burn-rs/burn-no-std-tests"
```

Comment:
Fix the repository url like the one with burn-common.

Reply:
ok

Reply:
OK. I'll fix this. I discovered other packages have this way too and sometimes incorrect URLs.